packages/core/admin/admin/src/pages/Auth/components/ResetPassword.tsx

```diff
@@ -1,73 +1,110 @@
-import React, { useState } from 'react';
+import * as React from 'react';
 
 import { Box, Button, Flex, Main, TextInput, Typography } from '@strapi/design-system';
-import { Form, Link } from '@strapi/helper-plugin';
+import { Link } from '@strapi/design-system/v2';
+import { Form, auth, translatedErrors, useFetchClient, useQuery } from '@strapi/helper-plugin';
 import { Eye, EyeStriked } from '@strapi/icons';
 import { Formik } from 'formik';
-import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
+import { useMutation } from 'react-query';
+import { NavLink, useHistory } from 'react-router-dom';
 import styled from 'styled-components';
+import * as yup from 'yup';
 
-import { Logo } from '../../../../components/UnauthenticatedLogo';
+import { ResetPassword } from '../../../../../shared/contracts/authentication';
+import { Logo } from '../../../components/UnauthenticatedLogo';
 import {
   Column,
   LayoutContent,
   UnauthenticatedLayout,
-} from '../../../../layouts/UnauthenticatedLayout';
-import FieldActionWrapper from '../FieldActionWrapper';
+} from '../../../layouts/UnauthenticatedLayout';
 
-const PasswordInput = styled(TextInput)`
-  ::-ms-reveal {
-    display: none;
-  }
-`;
+import { FieldActionWrapper } from './FieldActionWrapper';
 
-const ForgotPassword = ({ onSubmit, schema }) => {
-  const [passwordShown, setPasswordShown] = useState(false);
-  const [confirmPasswordShown, setConfirmPasswordShown] = useState(false);
+const REGISTER_SCHEMA = yup.object().shape({
+  password: yup
+    .string()
+    .min(8, translatedErrors.minLength)
```

Comment:
This is displaying as

image.png

We added the min param to that error message but do you remember how to pass in min so it is displayed?